zap/src/main/java/org/zaproxy/zap/model/Context.java

```diff
@@ -343,17 +343,18 @@ private static void validateRegexs(List<String> regexs) {
      * Validates that the given regular expression is not {@code null} nor invalid.
      *
      * @param regex the regular expression to be validated
-     * @throws IllegalArgumentException if the regular expression is {@code null}.
+     * @throws IllegalArgumentException if the regular expression is {@code null} or Empty.
```

Comment:
- The description of the method could be tweaked as well. `validateRegexs` should be updated accordingly.
- The empty should be lower case (same in the exception message).